geld.lua

```diff
@@ -21,66 +20,89 @@ if args.unit then
     if id then
         unit = df.unit.find(id)
     else
-        qerror("Invalid ID provided.")
+        qerror("Invalid unit ID provided.")
+        return
     end
 else
     unit = dfhack.gui.getSelectedUnit()
 end
 
 if not unit then
     qerror("Invalid unit selection.")
+    return
```

Comment:
`qerror` throws, so a `return` is not necessary. static analyzers don't understand this, though, so they sometimes give a warning.